forms/reefWatch/Underwater weeding survey.json

```diff
@@ -0,0 +1,512 @@
+{
+  "id": "668c7b7dac0eee3abb3cdf9a",
+  "dateCreated": "2024-07-09T10:04:03Z",
+  "minOptionalSectionsCompleted": 1,
+  "supportsSites": false,
+  "tags": [],
+  "lastUpdated": "",
+  "createdUserId": "56575",
+  "external": false,
+  "activationDate": null,
+  "supportsPhotoPoints": false,
+  "publicationStatus": "published",
+  "externalIds": null,
+  "gmsId": null,
+  "name": "Underwater weeding",
+  "sections": [
+    {
+      "collapsedByDefault": false,
+      "template": {
+        "modelName": "Underwater weeding",
+        "dataModel": [
+          {
+            "dataType": "text",
+            "name": "supervisor",
+            "indexName": "supervisor",
```

Comment:
indexName is only required if user has specifically asked to be able to facet on the field. Remove them otherwise.

Reply:
Also since supervisor is a free text field. It is likely user entry would not be a controlled vocabulary.